src/ephemeris/setup_data_libraries.py

```diff
@@ -76,7 +76,7 @@ def populate_items(base_folder_id, has_items):
                         folder_id=base_folder_id,
                         file_type=has_items['ext']
                     )
-                except:
+                except Exception:
```

Comment:
Can you either use `log.exception` or `exc_info=True` for the logging? this may contain important information

Reply:
Sure

Reply:
It occurs when the filetype is unknown on the galaxy server, that's the only example I can think of.